nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/pom.xml

```diff
@@ -39,5 +39,21 @@
             <version>1.16.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-standard-record-utils</artifactId>
+            <version>1.15.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-record-serialization-service-api</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-record</artifactId>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
On initial evaluation, adding these dependencies to `nifi-event-transport` does not seem like the right approach.  Other uses of this module do not need record-oriented modules, and in particular, the dependency on the `nifi-record-serialization-service-api` is problematic.  It seems better to avoid these dependencies entirely, and instead place the classes in another module.  This could be in `nifi-standard-processors`, where `ListenTCPRecord` currently exists, or for reuse, perhaps a new module named something like `nifi-record-event-transport`.